### PR TITLE
Match more functions

### PR DESCRIPTION
**Pull request by PokeCodec**
Redid because my OG PR was in bad shape

### CODE REVIEW

**Comment by PokeCodec** [comment]
This is as far as I could get. Ready for review!

File: include/sprite.h [hunk 1]
```diff
@@ -208,7 +208,7 @@ struct Sprite
 
     /*0x3E*/ u16 inUse:1;               //1
              u16 coordOffsetEnabled:1;  //2
-             u16 invisible:1;           //4
+             bool16 invisible:1;           //4
```
Comment (GriffinRichards) [inline include/sprite.h:211]: Alignment

File: src/pokedex_cry_screen.c [hunk 2]
```diff
@@ -7,7 +7,6 @@
 #include "ewram.h"
 #include "trig.h"
 #include "sound.h"
-
```
Comment (GriffinRichards) [inline src/pokedex_cry_screen.c:10]: Put newline back

File: src/battle/anim/flying.c [hunk 3]
```diff
@@ -555,1341 +555,293 @@ void sub_80DA410(struct Sprite *sprite)
         sprite->data[3] &= 0xFF;
     }
 
+    //The below if statement relies on overflow when cast to unsigned whenver pos1.x and pos2.x add up to less than 32.
```
Comment (GriffinRichards) [inline src/battle/anim/flying.c:558]: Typo, `whenver`

**Comment by PokeCodec** [comment]
Fixed!

**Pull request merged**